frontend/src/pages/MapPage.jsx

```diff
@@ -1,15 +1,22 @@
-import { useLoaderData } from "react-router-dom";
+import { useLoaderData, NavLink } from "react-router-dom";
 import LeafletMap from "../components/Game/Map/LeafletMap";
 
 export default function MapPage() {
   const artworks = useLoaderData();
   return (
-    <div className="lg:h-[85vh]">
-      <span className=" flex flex-row justify-center font-extrabold text-4xl md:text-5xl lg:text-6xl mb-4 text-primary mt-0">
+    <div className="lg:h-[88vh] ">
+      <span className=" flex flex-row justify-center  font-extrabold text-4xl md:text-5xl lg:text-6xl mb-4 text-primary mt-0">
         LA CARTE
       </span>
       <div className="">
         <LeafletMap dbartworks={artworks} />
+        <NavLink
+          to="/submitartwork"
+          className=" flex justify-center w-[20rem] mb-2 mx-auto cursor-pointer 
```

Comment:
a button wrapped in a Link is best suited

Reply:
I can rework after PR thanks